### PR TITLE
Major restructuring

### PR DESCRIPTION
**Pull request by clouserw**

### CODE REVIEW

**Comment by clouserw** [comment]
New sidebar structure is like this: 
image

This also adds a CONTRIBUTING file with some explanation.  The new structure is modeled off https://documentation.divio.com/introduction/

Once this lands we should revisit our documenation contents to make sure the voice and content is appropriate for its section.  In some places we may want to split files

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Thanks for contributing to our documentation.
+
+Our documentation is organized after the [divio system](https://documentation.divio.com/introduction/).  Please familiarize yourself with the system to understand where your new documetation fits in this repository.
```
Comment (LZoog) [inline CONTRIBUTING.md:3]: ~documetation~ documentation

File: docs/how-tos/creating-an-account-locally.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Creating an Account Locally
+---
+
+Once you've got FxA running locally you'll probably want to create an account that you can use for development and testing purposes. This process can be done almost exactly like you would in production:
+
+1. Visit the [root page](http://localhost:3030/), which should ask you to enter your email. Enter the email you want to use for the account.
+
+  If you're taken to `/signin` you can just click "Use a different account" to return to the email field.
```
Comment (LZoog) [inline docs/how-tos/creating-an-account-locally.md:9]: We could use a note here about or linking to other docs about how to create an account through the "Looking for Firefox Sync?" link.

File: docs/how-tos/managing-yarn-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## On Your Machine
+
+FxA uses [Yarn](https://yarnpkg.com/) to manage its Node dependencies. Follow the [Development Setup guide](./development-setup) for how to install Yarn and the `yarn` command on your machine.
+
+:::note
+Once upon a time FxA used [`npm`](https://docs.npmjs.com/cli/v7/commands/npm) to manage its dependencies, which means you may come across documentation or commands that still reference its usage. In most cases you should be able to freely replace the keyword `npm` with `yarn`, but it's worth learning about [the differences](https://www.sitepoint.com/yarn-vs-npm/#comparingnpmandyarncommands) between the two CLIs.
+:::
+
+We have a _lot_ of dependencies, and they're constantly updating, so be sure to occasionally run `yarn install` to stay up-to-date.
```
Comment (LZoog) [inline docs/how-tos/managing-yarn-dependencies.md:13]: Could be worth noting that if you don't have (or don't want) `FXA_AUTO_INSTALL=1` set, a warning will be displayed letting you know if `yarn.lock` differs, which I personally prefer 🤷‍♀️

File: docs/reference/interruptive-surveys.md [hunk 4]
```diff
@@ -1,9 +1,11 @@
 ---
-id: interruptive-surveys
-title: Running Interruptive Surveys in FxA
-sidebar_label: Running Surveys in FxA
+title: Interruptive Surveys
 ---
 
+:::caution
+This feature is being removed in [FXA-3880](https://mozilla-hub.atlassian.net/browse/FXA-3880)
```
Comment (LZoog) [inline docs/reference/interruptive-surveys.md:6]: I went ahead and specified in the ticket the documentation update is in ecosystem-platform

File: docs/reference/storybook-deploys-with-circleci.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Storybook Deploys with CircleCI
```
Comment (LZoog) [inline docs/reference/storybook-deploys-with-circleci.md:2]: We can probably remove a big chunk of [this section](https://github.com/mozilla/fxa/#storybook-in-circleci) on our root readme to not maintain two sets of docs. I think we can probably leave a few sentences and cross-link here for more, the screenshot on the repo home page is helpful IMO, but I'm fine with whatever.

File: docs/tutorials/development-setup.mdx [hunk 6]
```diff
@@ -0,0 +1,239 @@
+---
+title: Development Setup
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Getting Started
```
Comment (LZoog) [inline docs/tutorials/development-setup.mdx:8]: Not sure exactly how much repeating there is with our docs on the FxA front page (I'm trying to just skim this PR) but we should probably at least cross-link the link over there

**Review by LZoog — APPROVED** [review]
Briefly checked this out locally and didn't do much of a deep dive for review, I know you mostly copied existing stuff over so my content comments can obviously be done in a follow up and most aren't related to just refactoring, but yeah this is _noice_ :shipit:

File: docs/how-tos/managing-yarn-dependencies.md (hunk 3 above)
Comment (clouserw) [reply docs/how-tos/managing-yarn-dependencies.md:13]: I'll add that

**Pull request merged**